src/components/table/header/index.tsx

```diff
@@ -1,42 +1,16 @@
-import { Component, Event, EventEmitter, h, Host, Listen, Prop } from "@stencil/core"
+import { Component, h, Host } from "@stencil/core"
 
 @Component({
 	tag: "smoothly-table-header",
 	styleUrl: "style.css",
 	scoped: true,
 })
 export class TableHeader {
-	@Prop() name: string
-	@Prop({ mutable: true }) sortDirection?: "ascending" | "descending"
-	@Event() sort: EventEmitter<{ property: string; direction: "ascending" | "descending" }>
-	@Listen("click")
-	onClick(event: MouseEvent) {
-		this.sort.emit({
-			property: this.name,
-			direction: (this.sortDirection = this.sortDirection == "ascending" ? "descending" : "ascending"),
-		})
-		console.log("sort", this.name, this.sortDirection)
-		event.preventDefault()
-		event.stopPropagation()
-	}
 	render() {
```

Comment:
The `render` function does not need `Host`. Host is always on the object even if you don't specify.
The only reason you need to write `Host` is if you want to set any properties on host.

Can write the method as
``` tsx
render() {
  return <slot></slot>
}
```